dependencies.dot

```diff
@@ -9,17 +9,23 @@ digraph {
   fv3core [shape=oval, label="fv3core"]
   driver [shape=oval, label="driver"]
   physics [shape=oval, label="fv3gfs-physics"]
+  stencils [shape=oval, label="stencils"]
   util [shape=oval, label="fv3gfs-util"]
 
+
   pace -> fv3core
   pace -> physics
   pace -> util
   pace -> driver
+  pace -> stencils
```

Comment:
I expect pace will only indirectly depend on stencils - we probably won't instantiate its objects directly in a top-level example. However it's kind of odd in general that pace is in this file when it's not a subpackage, so this is fine either way.